proxy/envoy/config.go

```diff
@@ -226,15 +290,64 @@ func buildListeners(instances []*model.ServiceInstance,
 		hosts := make(map[string]VirtualHost, 0)
 		for _, proto := range []model.Protocol{model.ProtocolHTTP, model.ProtocolHTTP2, model.ProtocolGRPC} {
 			for _, svc := range lst.services[proto] {
-				host := buildHost(svc, OutboundClusterPrefix+svc.String(), suffix)
+				// if there is a routing rule for the host, use that to generate the route block
+				routes := make([]Route, 0)
+				// TODO: do we match on partial hostnames (foo.svc, foo.svc.namespace, etc. ?)
```

Comment:
is this implemented? this used to be supported and the test already exercises this functionality.